src/components/ControlledSelectInputField/ControlledSelectInputField.js

```diff
@@ -0,0 +1,83 @@
+import { makeStyles, MenuItem, TextField } from "@material-ui/core"
+import { Controller } from "react-hook-form"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    fontSize: theme.typography.fontSize,
+    fontFamily: theme.typography.fontFamily,
+
+    "& .MuiFormHelperText-contained": {
+      margin: "0",
+    },
+
+    "& .MuiOutlinedInput-input": {
+      padding: "6px 10px 6px 10px",
+      fontSize: theme.typography.fontSize,
+    },
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightBold,
+    paddingBottom: "4px",
+  },
+}))
+
+const ControlledSelectInputField = ({
+  control,
+  name,
+  label,
+  placeholder,
+  options,
+}) => {
+  const classes = useStyles()
+
+  const validateInput = (value) => {
+    if (!value) return false
+    if (value.length < 3) return false
+
+    return true
+  }
+
+  return (
+    <div className={classes.root}>
+      <span className={classes.label}>{label}</span>
+
+      <Controller
+        control={control}
+        name={name}
+        render={({ value, onChange }) => {
+          return (
+            <TextField
+              value={value}
+              placeholder={placeholder}
+              variant="outlined"
+              onChange={(event) => onChange(event.target.value)}
+              size="small"
+              select
```

Comment:
Can you actually use this package called `react-select` for the selector? Sorry I didn't mention it earlier, it's a much better and more powerful component for selectors